src/components/common/CreateUserWizard/StepUserName.tsx

```diff
@@ -36,62 +38,59 @@ const MSG = defineMessages({
 });
 
 interface UsernameInputProps {
-  username: string;
   disabled: boolean;
 }
-const UsernameInput = ({ username, disabled }: UsernameInputProps) => (
-  <>
-    <Input
-      appearance={{ theme: 'fat' }}
-      name="username"
-      label={MSG.label}
-      formattingOptions={{ blocks: [100] }}
-      disabled={disabled}
-    />
-    {username && isConfusing(username) && <ConfusableWarning />}
-  </>
+
+const formattingOptions = { blocks: [255] };
```

Comment:
The change to 255 chars was so the user name is the same length as the colony name ?

Reply:
Yes, I think when porting the UserCreationWizard we said we'd make them consistent, but after updating the username validation regex I forgot to update the formatting options here.